README.md

```diff
@@ -22,12 +22,13 @@ servers and syncing data across devices and applications.
 
 To develop remoteStorage-enabled apps, you need to have a
 remoteStorage-compatible storage account. We recommend
-[reStore](https://github.com/jcoglan/restore) for running a local test server.
```

Comment:
What page though? The "servers" page isn't just focused on developers, so I think the README that's contained in the repo is a good place to give that pointer. The next paragraph there points to the wiki.